src/wp-includes/theme.php

```diff
@@ -1852,11 +1852,13 @@ function _custom_background_cb() {
 		$color = false;
 	}
 
-	$type_attr = current_theme_supports( 'html5', 'style' ) ? '' : ' type="text/css"';
+	$handle = 'custom-background';
+	wp_register_style( $handle, false );
 
 	if ( ! $background && ! $color ) {
 		if ( is_customize_preview() ) {
-			printf( '<style%s id="custom-background-css"></style>', $type_attr );
+			wp_add_inline_style( $handle, '' );
```

Comment:
This won't work because `add_inline_style` short-circuits if `$code` is empty. So it looks like this will require providing a comment:

```suggestion
			wp_add_inline_style( $handle, '/*...*/' );
```